src/intersections.jl

```diff
@@ -135,6 +140,7 @@ include("intersections/boxes.jl")
 include("intersections/segments.jl")
 include("intersections/rays.jl")
 include("intersections/raysegment.jl")
+include("intersections/lineray.jl")
```

Comment:
Let's follow the naming pattern and use rayline.jl instead

Reply:
ah i had the naming convention to put the more general objects first, so regarding lines > rays > segments my plan was to use
`lineray`, `linesegment` and `raysegment`.

this stems from my intuition, that a line intersects a ray and a segment.
What do you think?

Reply:
It would have been a good convention but we already have RayBox, SegmentTriangle and other names that start with a object that is "less general" for some sense of generality to an object that is more general. I think the easiest approach now is to rename to make it compatible with the surrounding code.

Reply:
ok, I committed the requested changes